lightkurve/tests/test_lightcurve.py

```diff
@@ -1299,3 +1313,21 @@ def test_fill_gaps_after_normalization():
     assert lc2.flux[2].value == 1e6
     assert lc2.flux[2].unit == u.cds.ppm
     assert lc2.flux_err[2] == 0.1
+
+
+@pytest.mark.parametrize("new_col_val", [([2, 3, 4] * u_e_s),   # Quantity
+                                         (np.array([2, 3, 4])), # ndarray
+                                         ([2, 3, 4])            # list
+                                         ])
+def test_force_numeric_columns_as_quantity(new_col_val):
+    lc = LightCurve(time=[1, 2, 3])
+    lc['col1'] = new_col_val
+    assert isinstance(lc['col1'], Quantity)
```

Comment:
Note: the tests are failing here with `NameError: name 'Quantity' is not defined`.  This will be easy to fix!

Reply:
Fixed. It's embarrassing - I failed to notice the test error in my local build after the last round of test tweak.

Reply:
No worries!  That happens to me all the time!  That's why we have GitHub CI to protect us from ourselves 😅